consumer/consumer.go

```diff
@@ -47,7 +47,7 @@ func fetchSecretFromConjur(secretID string, conjurLogin string, conjurAPIKey str
 	fmt.Fprintf(os.Stderr, "Connecting to Conjur...\n")
 
 	// Receive the consumer conjur login and key
-	config := conjurapi.LoadConfig()
+	config, err := conjurapi.LoadConfig()
```

Comment:
It's probably worth actually handling the error that you get back. I.e.:

```suggestion
	config, err := conjurapi.LoadConfig()
	if err != nil {
		return "", err
	}
```